backend/LexBoxApi/GraphQL/CustomTypes/RefreshJwtOrgMembershipMiddleware.cs

```diff
@@ -0,0 +1,77 @@
+using HotChocolate.Resolvers;
+using LexBoxApi.Auth;
+using LexCore.Auth;
+using LexCore.Entities;
+
+namespace LexBoxApi.GraphQL.CustomTypes;
+
+public class RefreshJwtOrgMembershipMiddleware(FieldDelegate next)
+{
+    private const string REFRESHED_USER_KEY = "RefreshedJwtProjectMembership";
```

Comment:
Perhaps just "RefreshedJwtMemberships".

Reply:
Changed in commit 4af1ec9b. This constant should really be moved out of these two classes now, and moved into a shared class so that both classes can reference it. I don't see an obvious place to put it: should I create a new (static) class to store this constant in, or would you prefer for this to go in an existing class? Would welcome input from @hahn-kev here.